File: app/src/main/java/com/x7/steambadger/database/model/Badge.java
package com.x7.steambadger.database.model;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

import java.io.Serializable;

@DatabaseTable
public class Badge implements Serializable {

    @DatabaseField(generatedId = true)
    private Long id;
    @DatabaseField
    private int badgeId;
    @DatabaseField
    private String appId;
    @DatabaseField
    private int borderColor;
    @DatabaseField
    private int level;
    @DatabaseField
    private String text;
    @DatabaseField
    private String imageUrl;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public int getBadgeId() {
        return badgeId;
    }

    public void setBadgeId(int badgeId) {
        this.badgeId = badgeId;
    }

    public String getAppId() {
        return appId;
    }

    public void setAppId(String appId) {
        this.appId = appId;
    }

    public int getBorderColor() {
        return borderColor;
    }

    public void setBorderColor(int borderColor) {
        this.borderColor = borderColor;
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/com/dougmaitelli/steambadger/database/model/Badge.kt
package com.dougmaitelli.steambadger.database.model

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

import java.io.Serializable

@DatabaseTable
class Badge : Serializable {

    @DatabaseField(generatedId = true)
    var id: Long? = null
    @DatabaseField
    var badgeId: Int = 0
    @DatabaseField
    var appId: String? = null
    @DatabaseField
    var borderColor: Int = 0
    @DatabaseField
    var level: Int = 0
    @DatabaseField
    var text: String? = null
    @DatabaseField
    var imageUrl: String? = null
}
